UniversityofHelsinki/Styleguide
# Menu and footer improvements

## Description
- Changes background color of overlay, header and footer from black (```#222```) to pitch black (```#000```).
- Changes style of links in quick menu.
- Changes style of active and hovered main level and secondary level links in overlay navigation.

## How to test
1. Run ```gulp serve``` in styleguide root directory.
2. Open http://localhost:3000/#section-17-1.
3. Verify that Background color of overlay is pitch black (```#000```).
4. Verify active and hovered links get only underline effect.
5. Open http://localhost:3000/#section-15-1-2. Verify header has pitch black (```#000```) background and links get only underline as hover and active style.
6. Open http://localhost:3000/#section-15-2. Verify footer has pitch black (```#000```) background.

## Screenshots

![1](https://user-images.githubusercontent.com/3032567/38613002-1e779db4-3d91-11e8-97bb-1b4e2a8b949f.jpg)

![2](https://user-images.githubusercontent.com/3032567/38613008-20aeb108-3d91-11e8-8705-dd2a400b3f2e.jpg)

![3](https://user-images.githubusercontent.com/3032567/38613011-229e40fa-3d91-11e8-9cdd-953da2d54542.jpg)
